apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/db/DatabaseManagerTest.java

```diff
@@ -34,7 +40,8 @@
 class DatabaseManagerTest {
 
     private static String BASE_SUB_DIR = "unit-test-db";
-    private static String DB_FILE_NAME = "test_db";
+    private static String DB_FILE_NAME = "apl-blockchain";
```

Comment:
You can use TemporaryFolderExtension or in-memory DbProperties from DbTestData

Reply:
I suppose it we'll better to do after full merge both branches together.